libraries/core_libs/network/src/tarcap/packets_handlers/get_pbft_sync_packet_handler.cpp

```diff
@@ -32,6 +32,12 @@ void GetPbftSyncPacketHandler::process(const PacketData &packet_data,
   LOG(log_tr_) << "Received GetPbftSyncPacket Block";
 
   const size_t height_to_sync = packet_data.rlp_[0].toInt();
+  try {
+    pbft_syncing_state_->updatePeerAskingPeriod(packet_data.from_node_id_, height_to_sync);
+  } catch (const std::logic_error &e) {
```

Comment:
No need to use exception if your use case is basically checking return value of one specific function...  You could rarther check just returned bool flag